Makefile.common

```diff
@@ -82,6 +82,12 @@ endif
 # generic CFLAGS/LDFLAGS
 LDFLAGS += -L$(PREFIX_A)
 
+#add multilib directory to library search path
```

Comment:
please add space (` `) after the `#` for readability

Reply:
> LGTM, maybe apply readability fix.
> 
> Please note that with this change we could use -lphoenix (or -lc) for linking instead of manually providing the libphoenix.a to `ld`. Don't do it right now, IMHO we should move to linking via gcc so all of these problems would go away by design.

Thanks, space added and I've also created an issue for that.